advanced/Timeseries stitching demo.ipynb

```diff
@@ -0,0 +1,1200 @@
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1170316637448719</td>\n",
+       "      <td>heatSensor:985ccacb-1594-4ea0-9bd3-91986115a1c...</td>\n",
+       "      <td>heatSensorTimeSeries</td>\n",
+       "      <td>False</td>\n",
+       "      <td>degrees, celsius</td>\n",
+       "      <td>7049557553599949</td>\n",
+       "      <td>False</td>\n",
+       "      <td>A time series for a heat sensor</td>\n",
+       "      <td>1586264982766</td>\n",
+       "      <td>1586264982766</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "[<cognite.client.data_classes.time_series.TimeSeries object at 0x7f54ddbe0610>, <cognite.client.data_classes.time_series.TimeSeries object at 0x7f54c4cc19d0>]"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sensor_instance_1_id = assets.loc[assets[\"externalId\"] == sensor_instance_1_external_id].iloc[0][\"id\"]\n",
+    "sensor_instance_2_id = assets.loc[assets[\"externalId\"] == sensor_instance_2_external_id].iloc[0][\"id\"]\n",
+    "\n",
+    "sensor_1_timeseries_external_id = sensor_instance_1_external_id + \":timeseries\"\n",
+    "sensor_1_timeseries = TimeSeries(\n",
+    "    external_id=sensor_1_timeseries_external_id,\n",
+    "    asset_id=sensor_instance_1_id,\n",
+    "    name=\"heatSensorTimeSeries\",\n",
+    "    description=\"A time series for a heat sensor\",\n",
+    "    unit=\"degrees, celsius\"\n",
+    ")\n",
+    "sensor_2_timeseries_external_id = sensor_instance_2_external_id + \":timeseries\"\n",
+    "sensor_2_timeseries = TimeSeries(\n",
+    "    external_id=sensor_2_timeseries_external_id,\n",
+    "    asset_id=sensor_instance_2_id,\n",
+    "    name=\"heatSensorTimeSeries\",\n",
+    "    description=\"A time series for a heat sensor\",\n",
+    "    unit=\"degrees, celsius\"\n",
+    ")\n",
+    "client.time_series.create([sensor_1_timeseries, sensor_2_timeseries])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Populating TimeSeries\n",
+    "The **first sensor** breaks at 2017-10-15:14:35:47 and begins recording incorrect readings, 50-150 degrees higher than the acutal temperature. The **second sensor** is installed and begins recording correct data at 2017-10-15:14:41:42."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sensor_break_time = int(datetime(2017, 10, 15, 14, 35, 47).timestamp() * 1000)\n",
+    "sensor_replacement_time = int(datetime(2017, 10, 15, 14, 41, 42).timestamp() * 1000)\n",
+    "\n",
+    "sensor_1_start = sensor_break_time - (10 * 1000)\n",
+    "sensor_1_end = sensor_replacement_time + (10 * 1000)\n",
+    "sensor_1_data = lambda x: random.uniform(50, 55) if x < sensor_break_time else random.uniform(100,200)\n",
```

Comment:
I just wanted a sequence of data that I didn't have to hard code and that looked like it could have some from an organic source; but then for the values to suddenly be in a completely different range as if the sensor started producing bad data.